cranelift/codegen/src/isa/aarch64/inst.isle

```diff
@@ -2229,7 +2229,13 @@
       (let ((dst WritableReg (temp_writable_reg $I64)))
         (ConsumesFlags.ConsumesFlagsReturnsReg (MInst.CSet dst cond) dst)))
 
-;; Helper for constructing `csetm` instructions.
+;; Helper for constructing `cset` instructions.
+(decl cset_paired (Cond) ConsumesFlags)
```

Comment:
Can you add a little documentation here about how this is different from plain `cset` and why one would use it instead?